phpBB/includes/functions.php

```diff
@@ -921,6 +921,16 @@ function markread($mode, $forum_id = false, $topic_id = false, $post_time = 0, $
 
 		return;
 	}
+	else if ($mode == 'usermention')
```

Comment:
No, this is not what I told yesterday. You should not add a new ode to this method, as this method is already called on viewtopic.php. You should add your notification type to https://github.com/flash1452/phpbb/blob/cea4ff49d003fc5febf86e2353f930ad4f6bca2f/phpBB/includes/functions.php#L600 https://github.com/flash1452/phpbb/blob/cea4ff49d003fc5febf86e2353f930ad4f6bca2f/phpBB/includes/functions.php#L687 
https://github.com/flash1452/phpbb/blob/cea4ff49d003fc5febf86e2353f930ad4f6bca2f/phpBB/includes/functions.php#L799